Sources/ViewInspector/SwiftUI/ViewThatFits.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+internal extension ViewType {
+    struct ViewThatFits {}
+}
+
+@available(iOS 16.0, macOS 13.0, tvOS 16.0, *)
+extension ViewType.ViewThatFits: SingleViewContent {
+
+    static func child(_ content: Content) throws -> Content {
+        let view: Any = try {
+            guard let rootContent = try? Inspector.attribute(path: "_tree|content", value: content.view) else {
+                // A ViewThatFits View renders only one of its child Views based on the available horizontal space.
+                // This inspection returns all children that can potentially be picked by `ViewThatFits`.
+                // https://developer.apple.com/documentation/swiftui/viewthatfits
+                return try Inspector.attribute(path: "content", value: content.view)
```

Comment:
Can you help me understand this fallback? When would we expect this to be encountered?

Reply:
I see we had similar code in https://github.com/nalexn/ViewInspector/pull/228 but I want to make sure it's necessary here.